### ISSUE TITLE
UMD Formatter invalid ModuleId for globals

### ISSUE THREAD

**Issue by dweidenfeld** [babel/babel#1207]
Hi there,

I have a sample project (https://github.com/dweidenfeld/es6-to-umd/) using babel to convert es6 to umd.
The error occurs on the render library (you can see a non-working version in https://github.com/dweidenfeld/es6-to-umd/tree/invalid-global-module-id).

If I export the lib/render.js functions without default like
```javascript
export function render(content) {
	if (typeof window !== 'undefined') {
		$('.output').html(content);
	} else {
		Utils.log(content);
	}
}
```

and import it like
```javascript
import { render } from './lib/render';
```

the built render.js exports like
```javascript
global.render = mod.exports;
```

but the built main.js imports like
```javascript
factory(mod.exports, global.Calculator, global.Utils, global.libRender);
```

so it cannot find global.libRender because it is exported as global.render.
If I change the built code manually after the build to global.render everything works fine.

It seems to be a bug for me, because the only way to pass around this is (see master branch) to export function render als default and import it like
```javascript
import render from './lib/render';
```

If you have some other ideas how to fix it, please let me know :)

**Comment by sebmck** [comment]
It should be using the basename instead of the entire location.

**Comment by dweidenfeld** [comment]
Thank you for your quick response.
Do you know which version will include the fix?

**Comment by sebmck** [comment]
@dweidenfeld None yet. I'll push out a new version later today.

**Comment by sebmck** [comment]
Sorry for the delay. Fixed as of 5.1.0, thanks!
